lighthouse-core/lib/emulation.js

```diff
@@ -39,7 +38,7 @@ const DESKTOP_EMULATION_METRICS = {
 };
 
 // eslint-disable-next-line max-len
-const NEXUS5X_USERAGENT = 'Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3963.0 Mobile Safari/537.36 Chrome-Lighthouse';
+const MOTOG4_USERAGENT = 'Mozilla/5.0 (Linux; Android 6.0.1; Moto G (4)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3963.0 Mobile Safari/537.36 Chrome-Lighthouse';
```

Comment:
fwiw WPT adds `Build/MPJ24.139-64`

Reply:
hmm... I copied this from WPT

Reply:
like from a real phone WPT run? maybe the emulated UA is a little off 😆

Reply:
https://www.webpagetest.org/result/200115_VW_c15479a68affb254d3322ce865b0adf7/

click to the LH report and scroll to the bottom. I grabbed the host UA.

Reply:
yep I guess the WPT emulated UA is a bit off then!

Reply:
I assumed this is not emulating anything.

![image](https://user-images.githubusercontent.com/4071474/72472415-5e4f4a00-3799-11ea-9169-e2f0e51eb8a1.png)


How are you getting the UA w/ `Build/MPJ24.139-64`

Reply:
You are correct, the emulated one mistakenly contains `Build/MPJ24.139-64`

https://github.com/WPO-Foundation/webpagetest/blob/901d2906711c32577f9c8e19603cf491096fcbf8/www/settings/mobile_devices.ini#L11